runtime/binding-http/src/main/java/io/aklivity/zilla/runtime/binding/http/internal/stream/HttpServerFactory.java

```diff
@@ -6655,6 +6647,24 @@ private static Array32FW<HttpHeaderFW> initHeaders(
         return builder.build();
     }
 
+    private static Array32FW<HttpHeaderFW> initHeadersEmpty(
+            HttpConfiguration config,
+            String16FW status)
+    {
+        final Array32FW.Builder<HttpHeaderFW.Builder, HttpHeaderFW> builder =
+                new Array32FW.Builder<>(new HttpHeaderFW.Builder(), new HttpHeaderFW())
+                        .wrap(new UnsafeBuffer(new byte[64]), 0, 64)
+                        .item(h -> h.name(HEADER_STATUS).value(status));
+
+        final String16FW server = config.serverHeader();
+        if (server != null)
+        {
+            builder.item(h -> h.name(HEADER_SERVER).value(server));
+        }
```

Comment:
@jfallows Do you think we need to update `HEADER_SERVER` as well? In the earlier static declaration we don't have this attribute?

Reply:
Update: Removed `HEADER_SERVER` attribute from newly added method `initHeadersEmpty` as it was causing the related ITs to fail.